bundles/org.openhab.binding.vektiva/src/main/resources/ESH-INF/thing/smarwi.xml

```diff
@@ -15,7 +15,7 @@
 		</channels>
 
 		<config-description>
-			<parameter name="ip" type="text" required="true">
```

Comment:
I think you removed the wrong required=„true“ here. IP is required, refresh not.

Reply:
you are right. reverted.